dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -16,10 +36,111 @@ def index
     end
   end
 
+  # POST /api/v1/pd/workshops/1/enrollments
+  def create
+    @workshop = Pd::Workshop.find_by_id params[:workshop_id]
+    if @workshop.nil?
+      return render json: {submission_status: RESPONSE_MESSAGES[:NOT_FOUND]},
+        status: 404
+    end
```

Comment:
we expect `submissionStatus`, which we won't have if we just go with the ActiveRecord error. I could make a separate conditional in the UI that checks for a 404 status rather than `submissionStatus`, but I think this feels a little more consistent to me.

Reply:
I've been using `submissionStatus` as the indicator. 500 errors caused by the enrollment not saving will have the status `'error'`, but there could be unexpected 500 errors that won't get that `submissionStatus`. I can tell the UI to pay attention to the status codes instead of using the `submissionStatus` string.